src/server/service/kv.rs

```diff
@@ -624,6 +627,48 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
         ctx.spawn(future);
     }
 
+    fn kv_unsafe_cleanup_range(
```

Comment:
Why not just reuse `kv_delete_range`?

Reply:
Yep, looks like we don't need a new API now.

Reply:
In my understanding these two have different semantics, showned in `mvcc/reader.rs`. Maybe we can offer an API `UnsafeCleanMvccRange`, but implement it with optimized `delete_range`?